pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -573,8 +573,10 @@ type PodSandboxConfig struct {
 	// tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `protobuf:"bytes,7,rep,name=annotations" json:"annotations,omitempty" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
 	// Optional configurations specific to Linux hosts.
-	Linux            *LinuxPodSandboxConfig `protobuf:"bytes,8,opt,name=linux" json:"linux,omitempty"`
-	XXX_unrecognized []byte                 `json:"-"`
+	Linux *LinuxPodSandboxConfig `protobuf:"bytes,8,opt,name=linux" json:"linux,omitempty"`
+	// Optional custom image of pod's infra container
+	PodInfraContainerImage *string `protobuf:"bytes,9,opt,name=podInfraContainerImage" json:"podInfraContainerImage,omitempty"`
```

Comment:
I'm not sure whether we want to pass this through CRI. At least, currently infra container is a docker specific thing.

IIRC, we can only specify infra container image per-node not per-container, if so I feel like we should pass this to dockershim when creating docker service instead of passing through CRI.